src/legacy/core_plugins/vis_default_editor/public/components/controls/time_interval.tsx

```diff
@@ -86,32 +118,11 @@ function TimeIntervalParamEditor({
     </>
   );
 
-  const errors = [];
-
-  if (!isValid && value) {
-    errors.push(
-      i18n.translate('visDefaultEditor.controls.timeInterval.invalidFormatErrorMessage', {
-        defaultMessage: 'Invalid interval format.',
-      })
-    );
-  }
-
-  const onCustomInterval = (customValue: string) => {
-    const normalizedCustomValue = customValue.trim();
-    setValue(normalizedCustomValue);
-
-    if (normalizedCustomValue && isValidInterval(normalizedCustomValue, timeBase)) {
-      agg.write();
```

Comment:
I just removed `agg.write()` invocation since it's called anyway by a parent component. `DefaultEditorAgg` [calls](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/vis_default_editor/public/components/agg.tsx#L111) `makeLabel` for agg each time the component's rendered. And in Date Histogram aggregation inside `makeLabel` `writeParams` is [called](https://github.com/elastic/kibana/blob/master/src/plugins/data/public/search/aggs/buckets/date_histogram.ts#L79) which is the same as `agg.write`. So we don't need to call it here.

Reply:
I agree with you, that such calling is a spare thing here,
but this removal could be unsafe in case if `makeLabel` would be rewritten (hope this happen).

Reply:
I see, that's why I'd also add some functional tests. If in general these changes are acceptable, I'll add functional tests for broken cases.